docs/results.md

```diff
@@ -0,0 +1,376 @@
+GPLT Test Results
+=========================
+
+#### Table of Contents
+
+- [Background](#background)
+- [Performance](#performance)
+  * [Host directories](#host-directories)
+    + [Master](#master)  
+      * [atop measurements](#atop-measurements)
+      * [var logs](#var-logs)        
+    + [Metric](#metric)    
+      * [Gatling results](#gatling-results)
+        + [HTML report](#html-report)
+        + [Simulation log](#simulation-log)
+        + [JSON data](#json-data)
+  * [puppet-metrics-collector](#puppet-metrics-collector)
+  * [Tune settings](#puppet-metrics-collector)
+- [Scale](#scale)
+  * [Scale test report](#scale-test-report)
+  * [Beaker environment settings](#beaker-environment-settings)
+  * [Iterations](#iterations)
+    + [Iteration sub-directories](#iteration-sub-directories)
+      * [Host directories](#host-directories-1)
+        + [Master](#master-1)
+        + [Metric](#metric-1)
+  * [Gatling JSON data](#gatling-json-data)
+  * [pe_tune](#pe_tune)
+  * [puppet-metrics-collector](#puppet-metrics-collector-1)
+
+
+# Background
+[GPLT](https://github.com/puppetlabs/gatling-puppet-load-test) has two high-level test types: [Performance](#performance) and [Scale](#scale). 
+The test results are organized into separate directories for these test types.
+
+This document currently covers the Standard reference architecture. 
+[SLV-698](https://tickets.puppetlabs.com/browse/SLV-698) has been created to update it for the Large reference architecture.
+
+# Performance
+Performance tests consist of a single iteration of a scenario such as the [Apples2Apples](https://github.com/puppetlabs/gatling-puppet-load-test#apples-to-apples-performance-tests) and [Soak](https://github.com/puppetlabs/gatling-puppet-load-test#soak-performance-tests) tests. 
+The results for Performance test runs can be found in the [`results/perf`](https://github.com/puppetlabs/gatling-puppet-load-test/tree/master/results/perf) directory.
+
+The results for each Performance test run are contained within a separate timestamped sub-directory of the `results/perf` directory.
+The name of each result directory consists of the label `PERF_` to indicate a Performance run followed by a timestamp generated at the start of the run.
+For example:
+```
+results/perf
+└── PERF_1572446968
+```
+
+Performance result directories contain multiple files and sub-directories, each of which will be explained below.
+For example:
+  
+```
+results/perf/PERF_1572446968
+├── 20191030T144928Z-20191030T184941Z.tar.gz
+├── current_tune_settings.json
+├── end_epoch
+├── ip-10-227-0-214.amz-dev.puppet.net
```

Comment:
In the examples, please abstract the `master` and `metric` hostnames so that the reader doesn't have to refer back to this list.

%s/ip-10-227-0-214.amz-dev.puppet.net/master-hostname/g
%s/ip-10-227-0-248.amz-dev.puppet.net/metric-hostname/g

Reply:
I considered this previously but the downside of abstracting the hostnames is that it would mask the actual issue that GPLT users have when reviewing the results. By using actual hostnames in the examples I was trying to 'keep it real'. 

Since the following sections are labeled by role and the subsequent examples include the hostname it seems the reader should have context and shouldn't need to refer back to this list. However, if the impact to readability outweighs the value of providing a realistic example I will update accordingly.